### DISCUSSION TITLE
Set Iorth value (/PROP/SOL_ORTH) for LS-DYNA input

### DISCUSSION THREAD

**Discussion by dellowpl** [OpenRadioss/OpenRadioss#2945] [Q&A]
Hello,
Is it possible to explicitly set the value of Iorth within the /PROP/SOL_ORTH property when the input model is in LS-DYNA format?

**Comment by SyazwanSamad** [comment]
If direct translation is not available, I think you can mix both LS-DYNA and OpenRadioss keyword.
First delete the aforementioned *section_solid in LS-DYNA keyword. Create a new OpenRadioss include file, and define your /PROP/SOL_ORTH inside. Lastly, use *INCLUDE_RADIOSS inside the main LS-DYNA keyword.
